services/modules/helm-charts/charts/guardian-ipfs-client/templates/deployment.yaml

```diff
@@ -34,18 +34,54 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["/bin/sh"]
+          args: ["entrypoint.sh"]
           livenessProbe:
-            exec:
-              command:
-              - echo
-              - $HOME
-          resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+            {{- toYaml .Values.livenessProbe | nindent 12 }}
+          readinessProbe:
+            {{- toYaml .Values.readinessProbe | nindent 12 }}
+          resources: 
+            limits:
+              cpu: "{{ .Values.resources.cpu}}"
+              memory: "{{ .Values.resources.memory}}"
+            requests:
+              cpu: "{{ .Values.resources.cpu}}"
+              memory: "{{ .Values.resources.memory}}"
           envFrom:
             - configMapRef:
-                name: {{ include "guardian-ipfs-client.fullname" . }}-{{ .Release.Revision }}     
-          resources:
-            {{- toYaml .Values.resources | nindent 12 }}
+                name: {{ include "guardian-ipfs-client.fullname" . }}-{{ .Release.Revision }}
+            - secretRef:
+                name: {{ include "guardian-ipfs-client.fullname" . }}-{{ .Release.Revision }}
+            - configMapRef:
+                name: guardian-extensions-newrelic 
```

Comment:
Dynamic? especially as ENABLE_APM_NAME exists too

Reply:
Yes, we can make it dynamic to keep it clean but leave it there it will not impact anything

Reply:
Can you raise a ticket for this so we can track this as something to do?